engine/src/cmd/damageable.h

```diff
@@ -45,106 +44,107 @@ class Damageable : public DamageableObject
-  Damageable& operator=( const Damageable& ) = delete;
+    virtual ~Damageable() = default;
+    // forbidden
+    Damageable( const Damageable& ) = delete;
+    // forbidden
+    Damageable& operator=( const Damageable& ) = delete;
 
 public:
-  // We follow the existing convention of GetX for the actual health value
-  // because we are constrained by existing python interfaces, which cannot
-  // be easily changed.
-  const float GetHull() const {
-      return layers[0].facets[0].health.health;
-  }
+    // We follow the existing convention of GetX for the actual health value
+    // because we are constrained by existing python interfaces, which cannot
+    // be easily changed.
+    const float GetHull() const {
+        return layers[0].facets[0].health.health;
+    }
 
-  const float GetArmor(int facet = 0) const {
-      return layers[1].facets[facet].health.health;
-  }
+    const float GetArmor(int facet = 0) const {
+        return layers[1].facets[facet].health.health;
+    }
 
-  const float GetShield(int facet = 0) const {
-      return layers[2].facets[facet].health.health;
-  }
+    const float GetShield(int facet = 0) const {
+        return layers[2].facets[facet].health.health;
+    }
 
-  DamageableLayer& GetHullLayer() {
-      return layers[0];
-  }
+    DamageableLayer& GetHullLayer() {
+        return layers[0];
+    }
 
-  DamageableLayer& GetArmorLayer() {
-      return layers[1];
-  }
+    DamageableLayer& GetArmorLayer() {
+        return layers[1];
+    }
 
-  DamageableLayer& GetShieldLayer() {
-      return layers[2];
-  }
+    DamageableLayer& GetShieldLayer() {
+        return layers[2];
+    }
 
-  const float GetShieldRegeneration() const {
-      return layers[2].facets[0].health.regeneration;
-  }
+    const float GetShieldRegeneration() const {
+        return layers[2].facets[0].health.regeneration;
+    }
 
-  const float GetHullPercent() const
-  {
-      return layers[0].facets[0].health.Percent();
-  }
+    const float GetHullPercent() const
+    {
+        return layers[0].facets[0].health.Percent();
+    }
 
-  const float GetShieldPercent() const
-  {
-      return layers[2].facets[0].health.Percent();
-  }
+    const float GetShieldPercent() const
+    {
+        return layers[2].facets[0].health.Percent();
+    }
 
 
 
-  void ArmorData( float armor[8] ) const;
+    void ArmorData( float armor[8] ) const;
 
-  // Is the shield up from this position
-  bool ShieldUp( const Vector& ) const;
+    // Is the shield up from this position
+    bool ShieldUp( const Vector& ) const;
 
-  //applies damage from the given pnt to the shield, and returns % damage applied and applies lighitn
-  virtual float DealDamageToShield( const Vector &pnt, float &Damage );
-  virtual float DealDamageToHull( const Vector &pnt, float Damage );
+    // Currently available in case someone wants to call from python
+    // but there isn't an interface
+    float DealDamageToShield( const Vector &pnt, float &Damage );
```

Comment:
Is it safe to make this no longer virtual, or will that break backwards compatibility or anything?

Reply:
Excellent question. DealDamageToHull is no longer implemented in a subclass. Therefore, I don't think we need a virtual to specify which function is called. 

Please resolve issue if you think this is correct or continue on gitter if not.

Reply:
Saw the gitter discussion (@royfalk  thanks!) and pretty sure it does't need to be virtual.

Reply:
OK. Resolving.